tst/testextra/grpperm.tst

```diff
@@ -112,6 +112,15 @@ gap> Intersection(Group( (1,2,3), (4,5,6), (11,12,13), (11,12,14) ),
 Group(())
 gap> Intersection(Group((1,2), (3,4)), Group((3,4),(5,6)));
 Group( (3,4) )
+gap> Intersection(RightCoset(Group([ (1,2,3,4,5,6,7), (5,6,7) ]),(3,6)(4,7)),
+                  RightCoset(Group([ (1,2,3,4,5,6,8), (1,3,2,6,4,5), (1,6)(2,3)(4,5)(7,8) ]),(1,7,6,8,3,5)));
```

Comment:
Thanks. I did not realize this problem. So yes I should move those tests. But I realize there are several problems:
* I added the tests for Intersection of permutation groups (merged a few days ago) in testextra. So, I should move them to another file in testinstall. Because the same issue applies here.
* The RightCoset has almost no tests related to it. I think I should create a cset.tst in testinstall. 

Shall this be part of the same PR? Let me know what you think.

Reply:
Ideally, yes, this should be part of this PR, so that we can see from the Codecov reports which parts of the code are exercised.

Feel free to add a new file `tst/testinstall/coset.tst` or so and put your tests in there. There should ideally be also tests for (cosets of) groups which are not permutation groups, so that the code for those is also tested